feat: add external collaboration tool links (#1885)
gitpod-io/website: gitpod/docs/sharing-and-collaboration.md

> **A running Gitpod workspace really is your personal machine.**
>
> At present, this feature cannot be disabled. If this is something you are interested in, please [let us know in this issue](https://github.com/gitpod-io/gitpod/issues/6328).

## External Collaboration Plugins

With Gitpod you can also use the following third party integrations for collaboration:

> - [Code Together](https://www.codetogether.com/) - Works on VS Code Desktop, VS Code Browser and via JetBrains.

Author: For this, I had to create this file `/home/gitpod/.gitpod-code/argv.json` maybe we can already add it to workspaces? Unless Code Together fix the issue...  however @wgalanciak mentions it should work with Gitpod in https://github.com/Genuitec/CodeTogether/releases/tag/2022.1.3

Reviewer: There was also a [2022.1.4 release](https://github.com/Genuitec/CodeTogether/releases/tag/2022.1.4) meanwhile.

> - [Duckly](https://duckly.com/) - Duckly currently only works with VS Code Desktop (due to CORS constraints on gitpod.io).

Author: @galesky is looking into the CORS issue, maybe we should wait a day or two and see if they can resolve before updating the documentation on our side [[1](https://twitter.com/GaleskyLeonardo/status/1512479298682195968?s=20&t=wI0M4l9I65ZYX7_E3nbFLg)]

Reviewer 2: > maybe we should wait a day or two
That would be great!

I also want to double-check the Jetbrains support as it should work just fine.

If for some reason we see that this fix may take a while I'll ping here in the next couple of days and we can merge as is

Reviewer 3: Thanks for moving this along. I just wanted to also note that

> [Code Together](https://www.codetogether.com/) - Works on VS Code Desktop, VS Code Browser and via JetBrains.

could be expanded to include Eclipse (and Eclipse-based IDEs) as well with something like:

> [Code Together](https://www.codetogether.com/) - Works on VS Code Desktop, VS Code Browser and via JetBrains or Eclipse.

Reviewer 2: Hey @loujaybee, 

I've just published a new version of the extension (0.4.10). The install and CORS issues have been fixed!

Unfortunately, we are having issues with `JetBrains Gateway`. Let's mention only VS Code while we work on it.

Thank you for pointing those out. I've sent a review suggestion to this PR with the updated text

Reviewer 4: @loujaybee, I can confirm that I was able to install the duckly extension in the gitpod browser workspace, and got sharing between a desktop vscode and a browser based gitpod workspace. Thanks @galesky!

Reviewer: Thank you all for the input! We've updated the section with the suggested changes.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/418083/165943276-4893e6cf-6629-4c4c-8a36-2bffee5a8002.png">

> - [Duckly](https://duckly.com/) - Duckly currently only works with VS Code Desktop (due to CORS constraints on gitpod.io).

Reviewer 2: ```suggestion
- [Duckly](https://duckly.com/) - Works on VS Code Desktop and VS Code Browser.
```
